frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue

```diff
@@ -3313,7 +3324,7 @@ export default Vue.extend({
 
       this.update_smooth_canvas(this.label_settings.smooth_canvas)
 
-      this.$forceUpdate();
+      await this.$forceUpdate();
```

Comment:
makes initialization work correctly 

otherwise too heavy processing can have unexpected init problems in classes

e.g. canvas_mouse_tools being undefined (flaky) depending on time updating